hbase-server/src/main/java/org/apache/hadoop/hbase/master/MasterRpcServices.java

```diff
@@ -1744,10 +1742,13 @@ public CompactRegionResponse compactRegion(final RpcController controller,
       master.checkInitialized();
       byte[] regionName = request.getRegion().getValue().toByteArray();
       TableName tableName = RegionInfo.getTable(regionName);
+      // TODO: support CompactType.MOB
       // if the region is a mob region, do the mob file compaction.
       if (MobUtils.isMobRegionName(tableName, regionName)) {
         checkHFileFormatVersionForMob();
-        return compactMob(request, tableName);
+        //return compactMob(request, tableName);
+        //TODO: support CompactType.MOB
```

Comment:
This is remainder for developer, why do we need to log it?

Reply:
Because this code path is handling requests where previously a mob-only compaction would happen. Logging expressly that we don't do that for now, and where there's more info about why will help minimize surprise for operators.

Reply:
Fixed.